python/docs/source/conf.py

```diff
@@ -24,10 +24,22 @@
 copyright = u'2019, John langford et al'
 author = u'John langford et al'
 
+
+# Read version automatically from the version.txt.----------------------------
+def rev_replace(st, old, new, occurrence):
+    li = st.rsplit(old, occurrence)
+    return new.join(li)
+
+path = os.getcwd() # path of present directory
+
+# This makes sure that the path is correct after replacing only the last
+# occurrence of 'python/docs/source' in path by 'version.txt'
+ver_path = rev_replace(path, 'python/docs/source', 'version.txt', 1)
```

Comment:
Does this work on windows too? Perhaps Pathlib is a cross platform way to do this?

Reply:
I think this will work for windows too. As it is just dependent on python (`os library`) and `string` operations. Still, I will have a look at this and provide you with an update on this.

Reply:
The new change will work for windows too.